heartbeat/aws-vpc-move-ip

```diff
@@ -213,25 +224,33 @@ ec2ip_validate() {
 }
 
 ec2ip_monitor() {
-        MON_RES=""
+	MON_RES=""
+	if [ "${OCF_RESKEY_lookup_type}" == "NetworkInterfaceId"]; then
+		EC2_ID="$(ec2ip_get_instance_eni)"
+		RESOURCE_TYPE="interface"
+  	else
```

Comment:
There are still a few stragglers that start with spaces :)
```
$ grep -n '^ ' heartbeat/aws-vpc-move-ip 
231:  	else
234:  	fi
326:  	EC2_NETWORK_INTERFACE_ID="$(ec2ip_get_instance_eni)"
```

Reply:
Think it's my editor playing with me :(
Will fix this.. tnx